restapi/src/main/java/io/stargate/web/docsapi/resources/NamespacesResource.java

```diff
@@ -112,6 +116,34 @@ public Response getAllNamespaces(
         });
   }
 
+  @GET
+  @ManagedAsync
+  @ApiOperation(value = "View all built-in reserved functions")
+  @ApiResponses(
+      value = {
+        @ApiResponse(code = 200, message = "OK", response = BuiltInApiFunctionResponse.class),
+        @ApiResponse(code = 401, message = "Unauthorized", response = Error.class),
+        @ApiResponse(code = 403, message = "Forbidden", response = Error.class),
+        @ApiResponse(code = 500, message = "Internal Server Error", response = Error.class)
+      })
+  @Path("/{namespace-id: [a-zA-Z_0-9]+}/functions")
+  public Response getBuiltInFunctions(
+      @ApiParam(
+              value =
+                  "The token returned from the authorization endpoint. Use this token in each request.",
+              required = true)
+          @HeaderParam("X-Cassandra-Token")
+          String token,
+      @ApiParam(value = "The namespace to use for the request.", required = true)
+          @PathParam("namespace-id")
+          final String namespaceName,
```

Comment:
validation missing?

Reply:
All the rest of the endpoints in this file are missing any validation (what validation would be needed?)

Reply:
My bad, `namespaceName` is the path param, no validation needed there.. Will resolve..